pyproject.toml

```diff
@@ -1,24 +1,22 @@
 [tool.poetry]
 name = "griptape-chat"
-version = "0.2.0"
+version = "0.28.2"
```

Comment:
instead of referring to 0.28.2, there's a way to just say "use latest 0.28" and I think it might be `~=` but can't remember. This means we're pinned to 0.28, but always take the latest patches.